### ISSUE TITLE
some features on the app not working with handytech basic braille and microsoft narrator

### ISSUE THREAD

**Issue by rilygreat** [HabitRPG/habitrpg-ios#1339]
>  A new bug report from @Sonja1001-Theissen82:
> 
> When I open the Habitica app, I notice that it is unfortunately not so accessible that I can use it smoothly with my HandyTech Braille display "HandyTech Basic Braille 40 4.2" and my new screen reader "Microsoft Narator". How do I get to the input area for the tasks? And in the IOS app on my iPhone, I'm missing the button for "Synchronize" and I'm missing a simplified way to integrate Microsoft To Do and Habitica. The music shouldn't be so circuit-like, but orchestral. You should also be able to hear the applause. It's a shame that you can't click on the extras for the avatar with the Braille display. Please build buttons into the terms like "skin", "body", etc. We blind users don't work with the mouse, but with the keys on the keypad and the buttons on our Braille display. But that's about all I have to criticize. Thank you.

**Comment by saraolson** [comment]
looking into this, unfortunately with one developer for android + ios, and the size of the app
i don't think we'd be able to successfully accommodate this without a major rehaul and expertise in the tools. we continue to try to improve the app with VoiceOver, which should help improve the functionality of braille tools. 

unsure of the comment on the sounds here, perhaps the wrong theme was chosen?

in regards to microsoft todo, we don't plan to support integration with any other task management apps. 

i'm going to close this, but we'll reference it when looking into areas of voiceover improvements.
